web/packages/design/src/ResourceIcon/resourceIconSpecs.ts

```diff
@@ -0,0 +1,279 @@
+  orcasecurity: { dark: i.orcasecurityDark, light: i.orcasecurityLight },
+  'outreach.io': forAllThemes(i.outreachIo),
+
+  pagerduty: forAllThemes(i.pagerduty),
+  panther: { dark: i.pantherDark, light: i.pantherLight },
+  parallels: forAllThemes(i.parallels),
+  pingdom: forAllThemes(i.pingdom),
+  podigee: forAllThemes(i.podigee),
+  polleverywhere: forAllThemes(i.polleverywhere),
+  portswigger: forAllThemes(i.portswigger),
+  postgres: forAllThemes(i.postgres),
+  posthog: { dark: i.posthogDark, light: i.posthogLight },
+  productboard: forAllThemes(i.productboard),
+  prometheus: forAllThemes(i.prometheus),
+
+  qualified: forAllThemes(i.qualified),
+  quickbooks: forAllThemes(i.quickbooks),
+
+  rabbitmq: forAllThemes(i.rabbitmq),
+  rds: forAllThemes(i.rds),
+  redhat: forAllThemes(i.redhat),
+  redshift: forAllThemes(i.redshift),
+  ringlead: forAllThemes(i.ringlead),
+  rippling: { dark: i.ripplingDark, light: i.ripplingLight },
+
+  salesforce: forAllThemes(i.salesforce),
+  sanity: forAllThemes(i.sanity),
+  scim: { dark: i.scimDark, light: i.scimLight },
+  securecodewarrior: {
+    dark: i.securecodewarriorDark,
+    light: i.securecodewarriorLight,
+  },
+  semrush: forAllThemes(i.semrush),
+  sendgrid: forAllThemes(i.sendgrid),
+  sentry: { dark: i.sentryDark, light: i.sentryLight },
+  sequoia: { dark: i.sequoiaDark, light: i.sequoiaLight },
+  selfhosted: forAllThemes(i.database),
+  server: forAllThemes(i.server),
+  servicenow: forAllThemes(i.servicenow),
+  shopify: forAllThemes(i.shopify),
+  '6sense': { dark: i.sixsenseDark, light: i.sixsenseLight },
+  skype: forAllThemes(i.skype),
+  slab: forAllThemes(i.slab),
+  slack: forAllThemes(i.slack),
+  snowflake: forAllThemes(i.snowflake),
+  spacelift: { dark: i.spaceliftDark, light: i.spaceliftLight },
+  sparrow: forAllThemes(i.sparrow),
+  stripe: { dark: i.stripeDark, light: i.stripeLight },
+
+  tableau: forAllThemes(i.tableau),
+  terraform: forAllThemes(i.terraform),
+  torq: { dark: i.torqDark, light: i.torqLight },
+  'troops.ai': { dark: i.troopsAiDark, light: i.troopsAiLight },
+  twilio: forAllThemes(i.twilio),
+  twitter: { dark: i.twitterDark, light: i.twitterLight },
+
+  userorbit: { dark: i.userorbitDark, light: i.userorbitLight },
+
+  validity: forAllThemes(i.validity),
+  valimail: forAllThemes(i.valimail),
+  varicent: { dark: i.varicentDark, light: i.varicentLight },
+  vendr: forAllThemes(i.vendr),
+  vercel: { dark: i.vercelDark, light: i.vercelLight },
+  weavegitops: forAllThemes(i.weavegitops),
+
+  windows: { dark: i.windowsDark, light: i.windowsLight },
+  workshop: { dark: i.workshopDark, light: i.workshopLight },
+
+  youtube: forAllThemes(i.youtube),
+
+  zapier: forAllThemes(i.zapier),
+  zendesk: { dark: i.zendeskDark, light: i.zendeskLight },
+  zoom: forAllThemes(i.zoom),
+  zoominfo: forAllThemes(i.zoominfo),
+};
+
+export type ResourceIconName = keyof typeof resourceIconSpecs;
+
+/** All icon names, exported for testing purposes. */
```

Comment:
It's actually used in production code too.

Reply:
oops copy paste